osbuild/util/mnt.py

```diff
@@ -14,7 +14,7 @@ def mount(source, target, bind=True, ro=True, private=True, mode="0755"):
 
     args = []
     if bind:
-        args += ["--rbind"]
+        args += ["--bind"]
```

Comment:
Were we doing `rbind` things or not at all?

Reply:
I removed the second commit that does the unification and will open a new PR with that once it's done. It requires writing tests first to ensure I am not stupid (again) and change the semantics. But the first PR should be fine and is already useful as it will  ensure that we actually see proper error information from mount. There is a small risk as the exception changes from SubprocessCallError to RuntimeError but I don't think we specifically catch that in our code.